_monitoring-plugins/ad/index.md

```diff
@@ -133,33 +153,12 @@ If you stop or delete a detector, make sure to delete any monitors associated wi
 To see all the configuration settings for a detector, choose the **Detector configuration** tab.
 
 1. To make any changes to the detector configuration, or fine tune the time interval to minimize any false positives, go to the **Detector configuration** section and choose **Edit**.
-- You need to stop the detector to change its configuration. Confirm that you want to stop the detector and proceed.
+- You need to stop real-time or historical analysis to change its configuration. Confirm that you want to stop the detector and proceed.
```

Comment:
should we change to real-time *and* historical analysis? Nothing can be changed before you stop both of them.